src/usr/local/sbin/openvpn.attributes.sh

```diff
@@ -20,16 +20,53 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+lockfile="/tmp/ovpn_${dev}_${username}_${trusted_port}.lock"
+rulesfile="/tmp/ovpn_${dev}_${username}_${trusted_port}.rules"
+anchorname="openvpn/${dev}_${username}_${trusted_port}"
+
 if [ "$script_type" = "client-connect" ]; then
+	i=1
+	while [ -f "${lockfile}" ]; do
+		if [ $i -ge 30 ]; then
+			/bin/echo "Timeout while waiting for lockfile"
+			exit 1
+		fi
+
+		/bin/sleep 1
+		i=$(( i + 1 ))
+	done
+	/usr/bin/touch "${lockfile}"
+
+	/bin/cat "${rulesfile}" | sed "s/{clientip}/${ifconfig_pool_remote_ip}/g" > "${rulesfile}.tmp" && mv "${rulesfile}.tmp" "${rulesfile}"
```

Comment:
Full paths are missing from `sed` and `mv`

Reply:
Whoops, I created a new change instead of putting it here - sorry. :\

Reply:
```suggestion
	/bin/cat "${rulesfile}" | /usr/bin/sed "s/{clientip}/${ifconfig_pool_remote_ip}/g" > "${rulesfile}.tmp" && /bin/mv "${rulesfile}.tmp" "${rulesfile}"
```

Reply:
thanks!
I added it to the squashed commit